### PR TITLE
SSE support for file size and mdate

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/10779

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/container/sse.ts [hunk 1]
```diff
@@ -195,6 +196,16 @@ export const onSSEProcessingFinishedEvent = async ({
...
     }
 
+    /**
+     * Resource changed, fetching data
+     */
+    if (resource.etag !== sseData.etag) {
+      const { resource: updatedResource } = await clientService.webdav.listFilesById({
+        fileId: sseData.itemid
+      })
+      resourcesStore.upsertResource(updatedResource)
```
[the remark this answers is not in the document]
Comment (AlexAndBear) [reply packages/web-runtime/src/container/sse.ts:206]: But lets change something

File: packages/web-runtime/src/container/sse.ts [hunk 2]
```diff
@@ -195,11 +196,21 @@ export const onSSEProcessingFinishedEvent = async ({
...
+     */
+    if (resource.etag === sseData.etag) {
+      return resourcesStore.updateResourceField({
+        id: sseData.itemid,
+        field: 'processing',
+        value: false
+      })
+    }
+
+    const { resource: updatedResource } = await clientService.webdav.listFilesById({
```
Comment (kulmann) [inline packages/web-runtime/src/container/sse.ts:210]: shouldn't this be `getFileInfo` instead of `listFilesById`?

Comment (AlexAndBear) [reply packages/web-runtime/src/container/sse.ts]: oh yeah, we can do this now, since we also retrieve the space id, via the sse event

**Review by kulmann — APPROVED** [review]
Awesome! 💪

**Pull request merged**